makeflow/src/makeflow.c

```diff
@@ -1851,6 +1876,13 @@ if (enforcer && wrapper_umbrella) {
 		}
 	}
 
+	if(batch_queue_type == BATCH_QUEUE_TYPE_MESOS) {
+		batch_queue_set_option(remote_queue, "mesos-path", mesos_path);
+		batch_queue_set_option(remote_queue, "mesos-master", mesos_master);
+		batch_queue_set_option(remote_queue, "mesos-preload", mesos_preload);
+		batch_queue_set_feature(remote_queue, "batch_log_name", batchlogfilename);
```

Comment:
Is setting the batch log name specific to Mesos? This should be covered by the if statement just previous using the feature you set in batch_job_mesos.c